lang/python/python3-zeroconf/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-zeroconf
+PKG_VERSION:=0.21.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=zeroconf-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://pypi.python.org/packages/source/z/zeroconf/
+PKG_HASH:=5b52dfdf4e665d98a17bf9aa50dea7a8c98e25f972d9c1d7660e2b978a1f5713
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/zeroconf-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-zeroconf
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Multicast DNS Service Discovery Library
+  URL:=https://github.com/jstasiak/python-zeroconf
+  DEPENDS:= \
+	  +python3-light \
+	  +python3-requests \
+	  +python3-six \
+	  +python3-netifaces \
+	  +python3-ifaddr
```

Comment:
From [setup.py](https://github.com/jstasiak/python-zeroconf/blob/0.21.3/setup.py#L49-L52), it looks like only ifaddr is required? (six was removed when they [dropped support for Python 2 and 3.3](https://github.com/jstasiak/python-zeroconf/commit/f22f421e4e6bf1ca7671b1eb540ba09fbf1e04b1), [netifaces was replaced by ifaddr](https://github.com/jstasiak/python-zeroconf/commit/7c0500ee19869ce0e85e58a26b8fdb0868e0b142))

Reply:
Oops. Good catch! It should be corrected now as I committed the wrong Makefile. I have already 4 copies of it. :/